lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.PlanarSurface.rb

```diff
@@ -0,0 +1,256 @@
+        return previous_construction_map
+      end
+    end
+
+    # Check if the construction type was already created.
+    # If yes, use that construction.  If no, make a new one.
+
+    # for multi-building type - search for the surface wwr type
+    surface_std_wwr_type = wwr_building_type
+    new_construction = nil
+    type = [template, climate_zone, surf_type, stds_type, occ_type]
+    # Only apply the surface_std_wwr_type update when wwr_building_type has Truthy values
+    if !wwr_building_type.nil? && (surf_type == 'ExteriorWindow' || surf_type == 'GlassDoor')
+      space = planar_surface.space.get
+      if space.hasAdditionalProperties && space.additionalProperties.hasFeature('building_type_for_wwr')
+        surface_std_wwr_type = space.additionalProperties.getFeatureAsString('building_type_for_wwr').get
+      end
+      type.push(surface_std_wwr_type)
+    end
+
+    if previous_construction_map[type] && !previous_construction_map[type].iddObjectType.valueName.to_s.include?('factorGround')
+      new_construction = previous_construction_map[type]
+    else
+      new_construction = model_find_and_add_construction(planar_surface.model,
+                                                         climate_zone_set,
+                                                         surf_type,
+                                                         stds_type,
+                                                         occ_type,
+                                                         surface_std_wwr_type,
+                                                         wwr_info,
+                                                         planar_surface)
+      if !new_construction == false
+        previous_construction_map[type] = new_construction
+      end
+    end
+
+    # Assign the new construction to the surface
+    if new_construction
+      planar_surface.setConstruction(new_construction)
+      OpenStudio.logFree(OpenStudio::Debug, 'openstudio.standards.PlanarSurface', "Set the construction for #{planar_surface.name} to #{new_construction.name}.")
+    else
+      OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.PlanarSurface', "Could not generate a standard construction for #{planar_surface.name}.")
+      return previous_construction_map
+    end
+
+    return previous_construction_map
+  end
+
+  # Get appropriate construction object based on type of surface or subsurface
+  # @author: Doug Maddox, PNNL
+  # @param: surface_category [string] type of surface: this is not an OpenStudio string
+  # @param: surface_type [string] SubSurfaceType: this is an OpenStudio string
+  # @param: cons_set [object] DefaultSubSurfaceConstructions object
+  # @return: [object] Construction object
+  def get_default_surface_cons_from_type(surface_category, surface_type, cons_set)
```

Comment:
This seems to be identical to the original version of this method. Can we remove it from the prm.PlanarSurface.rb?